osversion/platform_compat_windows.go

```diff
@@ -0,0 +1,71 @@
+package osversion
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+// List of stable ABI compliant
+// ltsc releases
+// Note: List must be sorted
+var compatLTSCReleases = []uint{
+	V21H2Server,
+}
+
+// Checks if given host and guest OS versions are compatible.
```

Comment:
Function doc strings should start with the function name. e.g. `AreHostAndGuestCompatible checks...`